lib/vagrant-registration.rb

```diff
@@ -15,6 +15,8 @@ def self.source_root
       @source_root ||= Pathname.new(File.expand_path("../../", __FILE__))
     end
 
+    # Temporally load the extra capability files for Fedora
+    load(File.join(self.source_root, 'plugins/guests/fedora/plugin.rb'))
```

Comment:
I think that is kind of anti pattern, this is temp. solution until we get guest changes to default vagrant plugins.